hypothesis-python/tox.ini

```diff
@@ -95,6 +95,13 @@ deps =
 commands =
     python -m pytest tests/pandas -n2
 
+[testenv:pandas23]
+deps =
+    -r../requirements/test.txt
+    pandas~=0.22.0
```

Comment:
Why are we installing pandas 0.22 in a 0.23 test?

Reply:
late-night typo 😞, nothing to see here...